src/getHumanAge.test.js

```diff
@@ -7,4 +7,36 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('dogs years and cats years are equal 0', () => {
```

Comment:
```suggestion
  test('dogs years and cats years are equal 0', () => {
```
it's not a good name for tests. We need to write something like - should return something when something

Reply:
fixed